docs/basics/control_flow_commands.md

```diff
@@ -0,0 +1,117 @@
+---
+icon: material/arrow-down-right
+search:
+    boost: 4
+---
+# :material-arrow-down-right: Control Flow Commands
+
+Control flow commands allow you to set step through the code, stop execution and resume it at your pleasure. 
+
+## :material-ray-end-arrow: Stepping
+A basic feature of any debugger is the ability to step through the code. **libdebug** provides several methods to step, some of which will be familiar to users of other debuggers.
+
+### :material-debug-step-into: Single Step
+The `step()` command executes the instruction at the instruction pointer and stops the process. When possible, it uses the hardware single-step feature of the CPU for better performance.
+
+!!! ABSTRACT "Function Signature"
+    ```python
+    d.step()
+    ```
+
+### :material-debug-step-over: Next
+The `next()` command executes the instruction at the instruction pointer and stops the process. If the instruction is a function call, it will execute the whole function and stop at the next instruction. In other debuggers, this command is known as "step over".
+
+Please note that the `next()` command resumes the execution of the program if the instruction is a function call. This means that the debugger can encounter [stopping events](../../stopping_events/stopping_events) in the middle of the function, causing the command to return before the function finishes.
+
+!!! ABSTRACT "Function Signature"
+    ```python
+    d.next()
+    ```
+
+!!! WARNING "Damn heuristics!"
+    The `next()` command uses heuristics to determine if the instruction is a function call and to find its end. This means that the command may not work as expected in some cases (e.g. tail-calls, non-returning calls, broken stack frame).
+
+### :material-debug-step-over::material-debug-step-over: Step Until
+
+The `step_until()` command executes single steps until a specific address is reached. Optionally, you can also limit steps to a maximum count (default value is -1, meaning no limit).
+
+!!! ABSTRACT "Function Signature"
+    ```python
+    d.step_until(position, max_steps=-1, file='hybrid') 
+    ```
+
+The file parameter can be used to specify the choice on relative addressing. Refer to the [memory access](../memory_access/#absolute-and-relative-addressing) section for more information on addressing modes.
+
+## :material-step-forward: Continuing
+
+The `cont()` command continues the execution.
+
+!!! ABSTRACT "Function Signature"
+    ```python
+    d.cont()
+    ```
+
+!!! INFO "Script execution"
+    Please note that, after resuming execution of the tracee process, the script will continue to run. This means that the script will not wait for the process to stop before continuing with the rest of the script. If the next command is a **libdebug** command that requires the process to be stopped, the script will then wait for a [stopping event](../../stopping_events/stopping_events) before executing that command.
```

Comment:
This should be clarified before, in earlier sections

Reply:
[9e99adc](https://github.com/libdebug/libdebug/pull/129/commits/9e99adc7c4f740849576a6ec29d1c746c4e2a5ef)